test/catFile.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var rimraf = require('rimraf');
+var should = require('should');
+var gutil = require('gulp-util');
+var it = require('mocha').it;
```

Comment:
`it` is inferred by scope through mocha, refer to the other tests. It should not be required